content/en/users/check-in/vos/_index.md

```diff
@@ -282,7 +282,7 @@ From the steps defined [above](#managing-cou-admin-members):
 
    ![Group members tab](./check-in-group-members-tab.png)
 
-## VO membership API
+## VO membership API v1 (DEPRECATED)
```

Comment:
Keep VO membership API as H2 level and introduce v2 H3 level and v1 H3 level (deprecated)